lms/templates/cme_register.html

```diff
@@ -378,78 +363,67 @@ <h3 class="message-title">Welcome ${extauth_id}</h3>
        
         <li class="field" style="display: none;" id="sub-specialty-container">
           <label for="sub_specialty">Sub-Specialty</label>
-          <select id="sub_specialty" name="sub_specialty" class="controller">
+          <select id="sub_specialty" name="sub_specialty">
             <option value="">--</option>
           </select>
          
-          <div class="field controlled" id="sub_specialty_free_container" style="display: none;">
-           <br />
-           <input id="sub_specialty_free" type="text" name="sub_specialty_free" title="Please enter your Sub-Specialty here." placeholder="Please enter your Sub-Specialty here." />
-          </div>
         </li>
        
-        <div id="sub_specialty-Cardiology" class="sub_specialty-container" style="display: none;">
+        <div id="sub_specialty-Cardiovascular_Health" class="sub_specialty-container" style="display: none;">
           <option value="">--</option>
-          %for code, sub_specialty in sub_specialty_choices['Cardiology']:
+          %for code, sub_specialty in sub_specialty_choices['Cardiovascular_Health']:
             <option value="${code}">${sub_specialty}</option>
           %endfor
         </div>
-       
+
         <div id="sub_specialty-Internal_Medicine" class="sub_specialty-container" style="display: none;">
           <option value="">--</option>
           %for code, sub_specialty in sub_specialty_choices['Internal_Medicine']:
             <option value="${code}">${sub_specialty}</option>
           %endfor
         </div>
-       
+
         <div id="sub_specialty-Obstetrics_Gynecology" class="sub_specialty-container" style="display: none;">
           <option value="">--</option>
-          %for code, sub_specialty in sub_specialty_choices['Obstetrics_Gynecology']:
+          %for code, sub_specialty in sub_specialty_choices['Obstetrics_&_Gynecology']:
```

Comment:
CME currently stores account information with ampersands. Account creation is handled in views.py, which receives the code with underscores and ampersands. We keep this to avoid inconsistencies in the database.